### PR TITLE
Remove templating dependency

### PR DESCRIPTION
**Pull request by pyrech**
This PR removes the dependency on symfony/templating component and uses Twig directly
Ref #200

### CODE REVIEW

**Comment by sod** [comment]
Generating json with a template engine seems weired alltogether.

in particular:
* https://github.com/willdurand/BazingaJsTranslationBundle/blob/master/Resources/views/config.json.twig
* https://github.com/willdurand/BazingaJsTranslationBundle/blob/master/Resources/views/getTranslations.json.twig

**Comment by monteiro** [comment]
I know that symfony wanted to use Twig by default because of the speed. @stof can you confirm this? Is it a good practice to replace the template engine for twig for speed?

**Comment by pyrech** [comment]
Yep, using Twig directly instead of the whole templating component is probably a small optimization in terms of speed because of the many classes avoided.
But at first I opened this PR just because when using Flex, the bundle installation failed and forced me to install and configure the templating component even if not using it in my app

File: composer.json [hunk 1]
```diff
@@ -15,8 +15,8 @@
         "symfony/finder": "~2.7|~3.1",
         "symfony/console": "~2.7|~3.1",
         "symfony/intl": "~2.7|~3.1",
-        "symfony/templating": "~2.7|~3.1",
-        "symfony/translation":  "~2.7|~3.1"
+        "symfony/translation":  "~2.7|~3.1",
+        "twig/twig": "~1.0|~2.0"
```
Comment (stof) [inline composer.json]: I suggesting depending on TwigBundle instead (as this bundle actually depends on services defined in the bundle, not just on Twig)

Comment (pyrech) [reply composer.json]: Done :+1:

**Comment by monteiro** [comment]
@stof thanks for the clarification.

**Comment by monteiro** [comment]
@pyrech let me see what are happening with the tests, since in master they failed as well.

**Comment by monteiro** [comment]
@pyrech can you plz rebase with master so we can have the tests green? Thanks!

**Comment by pyrech** [comment]
Done :wink:

**Comment by monteiro** [comment]
@pyrech it seems that the tests are failing for PHP 5.3 :( Since in master passes, is it a problem with this PR?

**Comment by pyrech** [comment]
Hum, looks like a problem with translation ordering but no idea where could it come from :confused:

**Comment by monteiro** [comment]
@pyrech tested locally with php5.3 and it worked 😢 travis not.

**Comment by monteiro** [comment]
@pyrech can you rebase with master. I have dropped php 5.3 support.

**Comment by pyrech** [comment]
Done, sorry for the delay

**Comment by monteiro** [comment]
@pyrech thanks!

**Pull request merged**